ts/receiver/closedGroups.ts

```diff
@@ -885,6 +886,40 @@ export async function createClosedGroup(groupName: string, members: Array<string
   convo.updateLastMessage();
 
   // Send a closed group update message to all members individually
+  const groupInviteResults = await sendToGroupMembers(
+    listOfMembers,
+    groupPublicKey,
+    groupName,
+    admins,
+    encryptionKeyPair,
+    dbMessage
+  );
+
+  const allInvitesSent = groupInviteResults.every(result => result === true);
+  if (!allInvitesSent) {
+    window.confirmationDialog({
+      title: 'Group Invite Failed',
+      message: 'Unable to successfully invite all group members',
+    });
+  }
+
+  await forceSyncConfigurationNowIfNeeded();
```

Comment:
we want to keep the 
`addClosedGroupEncryptionKeyPair` and the `SwarmPolling.getInstance().addGroupId` in this function rather than in the called `sendToGroupMembers`. Because if the sending fails for some members, we don't want to subscribe to the group pubkey neither save the keypair to the db.


also, the keypair might be saved several times depending on how you handle the confrimationDialog, because `addClosedGroupEncryptionKeypair` does not check for duplicates.

you have to call `  const isKeyPairAlreadyHere = await isKeyPairAlreadySaved(groupPublicKey, newKeyPairInHex);`